src/view/com/modals/Waitlist.tsx

```diff
@@ -19,7 +19,7 @@ import {useTheme} from 'lib/ThemeContext'
 import {ErrorMessage} from '../util/error/ErrorMessage'
 import {cleanError} from 'lib/strings/errors'
 
-export const snapPoints = ['80%']
+export const snapPoints = ['60%']
```

Comment:
Let's keep it at 80% for smaller phones or use a ScrollView

Reply:
Ah right. I tested large fonts but didnt think about smaller phones.